recipe/meta.yaml

```
@@ -10,40 +10,62 @@ source:
   sha256: bd94dd61edae3b23e43e70bbbfbada06e6d2f7fbac292ed1dbcfcb3afcca929a
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # because of restraint on pandas <2 missing for py312
+  skip: True  # [py>311]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
     - setuptools-scm
+    - wheel
   run:
+    - python
     - importlib-metadata
```

```suggestion
    - importlib-metadata  # [py<38]
```
see https://github.com/jamalsenouci/causalimpact/blob/v0.2.6/setup.cfg#L35